docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md

```diff
@@ -181,38 +181,6 @@ The following list contains role-based access control actions.
 { .no-spacing-list }
 <!-- prettier-ignore-end -->
 
-### Grafana OnCall action definitions (beta)
```

Comment:
nit: removing this, as it's duplicating info that exists [here](https://grafana.com/docs/grafana-cloud/alerting-and-irm/irm/oncall/manage/user-and-team-management/#manage-users-and-teams-for-grafana-oncall).